admin-dev/themes/new-theme/js/pages/product/combination/bulk.ts

```diff
@@ -37,4 +37,35 @@ $(() => {
   ]);
   new ImageSelector();
   new QuantityModeSwitcher();
+
+  const priceExcludedTaxId = 'bulk_combination_price_price_tax_excluded';
```

Comment:
So far we've taken the habit of putting those identifiers in mapping files, see [this file](https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/js/pages/specific-price/specific-price-map.ts) for example